src/Microsoft.Crank.Controller/JobConnection.cs

```diff
@@ -1218,6 +1218,11 @@ public async Task DownloadFileAsync(string file, string output)
                 var uri = Combine(_serverJobUri, "/download?path=" + HttpUtility.UrlEncode(file));
                 Log.Verbose("GET " + uri);
 
+                if (file.StartsWith("~/") || file.StartsWith("~\\"))
```

Comment:
Does `~/` work when the path has a `*`?

Reply:
Filenames here don't have `*`, the controller invokes a /list first when there is stars. Then invokes individual calls to /download